src/components/ManaCalculator/types.ts

```diff
@@ -42,12 +41,13 @@ export interface ManaCalculatorProps {
     performanceFactor: number;
   };
   stake: number;
+  network: NetworkType
 }
 
 export enum CongestionType {
-  LOW = RMC_LOW_CONGESTION,
-  MEDIUM = RMC_STABLE_CONGESTION,
-  HIGH = RMC_EXTREME_CONGESTION,
+  LOW = 100000,
```

Comment:
What was the reason for this change? My initial idea was to have all parameters in one place in case we ever need to change this constants 🤔 
Maybe then let's move the whole enum?

Reply:
It was giving me a circular dependency error iirc, utils.ts depeneding on types.ts and type.ts depending on utils.ts. I will split this again into an enum and constants in the refactor PR

Reply:
@Dr-Electron its just a temporary change, not ideal, but to speed up the PRs we decided on fixing the circular dependencies on the refactor PR which is coming

Reply:
Ok sounds good 👍